### PR TITLE
Fix for python path issues due to wrong version installed in docker-o…

### PR DESCRIPTION
**Pull request by NyakudyaA**
Fix for #52 
* It seems ubuntu:latest tag somehow defaults to python3 or installs python 3. 
* Migrate download.py in docker-osm-update to python3

### CODE REVIEW

**Comment by Gustry** [comment]
Thanks @NyakudyaA I will have a look.
I migrated docker-osm to python3 in a branch, but somehow I got an issue. I will try to remember what it was.

**Comment by NyakudyaA** [comment]
Great @Gustry  I only upgraded the download.py from docker-osmupdate and made other small tweaks to get it working

**Comment by Gustry** [comment]
@NyakudyaA Are you using your PR in production or in staging? Is-it working fine?

File: docker-compose.yml [hunk 1]
```diff
@@ -1,96 +1,94 @@
...
+    hostname: db
+    environment:
+      - POSTGRES_USER=docker
+      - POSTGRES_PASS=docker
+      - POSTGRES_DBNAME=gis
+    volumes_from:
+      - storage
+  # Uncomment to use the postgis database from outside the docker network
+    ports:
+      - "35432:5432"
```
Comment (Gustry) [inline docker-compose.yml]: Can you comment these lines according to L23 for security reason?

File: docker-compose.yml [hunk 2]
```diff
@@ -1,96 +1,94 @@
...
-    - POSTGRES_PASS=docker
-    - POSTGRES_DBNAME=gis
-  volumes_from:
-    - storage
-# Uncomment to use the postgis database from outside the docker network
-#  ports:
-#   - "5432:5432"
+services:
+  storage:
+    image: ubuntu:latest
```
Comment (Gustry) [inline docker-compose.yml]: BTW, can you switch to ubuntu:bionic? The latest LTR from ubuntu

Comment (NyakudyaA) [reply docker-compose.yml]: Have done so

File: docker-compose.yml (hunk 1 above)
Comment (NyakudyaA) [reply docker-compose.yml]: I have commented them out

**Comment by NyakudyaA** [comment]
Yes, I am using this in staging.
healthy

**Comment by Gustry** [comment]
Ok cool!

**Pull request merged**